extensions/amp-story/0.1/amp-story.js

```diff
@@ -478,6 +478,16 @@ export class AmpStory extends AMP.BaseElement {
       this.onKeyDown_(e);
     }, true);
 
+    // keep track of page-id in browser history API
+    this.storeService_.subscribe(StateProperty.CURRENT_PAGE_ID, pageId => {
+      const history = this.win.history;
+      if (history.pushState) {
+        history.pushState({
+          ampStoryPageId: pageId,
+        }, 'amp-story', '');
```

Comment:
The second parameter is the title of the page. Browsers don't support it for now, but we don't need to set it to 'amp-story'. I think it's more future proof to set it to null, in case browsers decide to support it.

Third parameter is the new URL and is optional, it's safer to skip it. Setting it to ``/`` wipes the URL on Chrome, I wouldn't be surprised if some browsers understand an empty string as "please wipe the URL".

Reply:
talked offline, omitted third param, second param changed to empty string